### PR TITLE
Add support file to all repositories

### LINKED ISSUE #3: Add support file to all repositories
All repositories under the Keycloak organization should have a common support file.

This should be done by adding a default SUPPORT.md file to the https://github.com/keycloak/.github repository.

### PR DESCRIPTION
**Pull request by stianst**
Closes #3

### CODE REVIEW

**Review by hmlnarik — APPROVED** [review]

**Review by mposolda — COMMENTED** [review]
Does also link to the Community page belongs here https://www.keycloak.org/community ? The page contains bunch of other links with some additional stuff (EG. how to report security issue, bugs, how to contribute etc). Or is it out of scope?

**Comment by stianst** [comment]
> Does also link to the Community page belongs here https://www.keycloak.org/community ? The page contains bunch of other links with some additional stuff (EG. how to report security issue, bugs, how to contribute etc). Or is it out of scope?

I'm planning on revamping that page soon:
https://github.com/keycloak/keycloak-web/issues/205

The files in this repo should be consistent with that page though (eventually)

**Comment by pedroigor** [comment]
@stianst Does it make sense to have something like:

```
For any security-relevant issues please consider looking at our security guidelines. 
```

Perhaps make more explicit that security-relevant issues should not go over any public channel.

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Need help with Keycloak?
+
+Welcome to the Keycloak community. If you have some questions or need some help we are here to help, please be respectful and also remember that we simply may not have time or capacity to answer every question.
```
Comment (abstractj) [inline SUPPORT.md]: I would suggest rephrasing to something like: "If you have some questions or need some help we are here to support you...", or "We are here to help you, please be respectful..."

**Review by abstractj — COMMENTED** [review]
Approving it with a minor comment, but it's not a deal-breaker to merge.

**Comment by mposolda** [comment]
> > Does also link to the Community page belongs here https://www.keycloak.org/community ? The page contains bunch of other links with some additional stuff (EG. how to report security issue, bugs, how to contribute etc). Or is it out of scope?
> 
> I'm planning on revamping that page soon: [keycloak/keycloak-web#205](https://github.com/keycloak/keycloak-web/issues/205)
> 
> The files in this repo should be consistent with that page though (eventually)

Ok, since it is considered how to point users to this page, I am approving this.

**Review by mposolda — APPROVED** [review]

**Review by thomasdarimont — APPROVED** [review]

**Review by pedroigor — APPROVED** [review]

**Comment by stianst** [comment]
> @stianst Does it make sense to have something like:
> 
> ```
> For any security-relevant issues please consider looking at our security guidelines. 
> ```
> 
> Perhaps make more explicit that security-relevant issues should not go over any public channel.

Fully agree, added a sentence on this

File: SUPPORT.md (hunk 1 above)
Comment (stianst) [reply SUPPORT.md]: Updated

**Pull request merged**